helm/values.yaml

```diff
@@ -0,0 +1,39 @@
+common:
+  fullnameOverride: "pr-custom-review"
+  image:
+    repository: paritytech/pr-custom-review
+  serviceAccount:
+    create: false
+  podSecurityContext:
+    fsGroup: 1000
+    runAsNonRoot: true
+    runAsUser: 1000
+    runAsGroup: 1000
+  env:
+    LOG_FORMAT: json
+    PORT: 3000
+  secrets:
```

Comment:
```suggestion
  secrets:
```
```suggestion
```

Reply:
https://gitlab.parity.io/parity/mirrors/pr-custom-review/-/jobs/1579390 is the job running for this change

Reply:
https://gitlab.parity.io/parity/mirrors/pr-custom-review/-/jobs/1579265 timed out and https://gitlab.parity.io/parity/mirrors/pr-custom-review/-/jobs/1579390 as well

> Error: release pr-custom-review failed, and has been uninstalled due to atomic being set: timed out waiting for the condition

Reply:
Yeah. `secret "pr-custom-review" not found: CreateContainerConfigError` need to cut more code :)